app/tests/unit/components/validators/validators.customValidators.spec.js

```diff
@@ -59,4 +60,28 @@ describe('customValidators.docGen', () => {
     expect(Array.isArray(result)).toBeTruthy();
     expect(result.length).toEqual(0);
   });
+
+  it('should return an error array when file type conversion is not supported', async () => {
+    body = {
+      contexts: [{
+        x: 1,
+        y: 2
+      }],
+      template: {
+        content: 'ZHNmc2Rmc2RmZHNmc2Rmc2Rmc2Rm',
+        contentEncodingType: 'base64',
+        contentFileType: 'docx',
+        outputFileType: 'ppt',
+      }
+    };
+
+    const result = await customValidators.docGen(body);
+
+    expect(result).toBeTruthy();
+    expect(Array.isArray(result)).toBeTruthy();
+    expect(result.length).toEqual(1);
+    expect(result[0].message).toMatch('Unsupported');
```

Comment:
It's probably better to match on a more complete string than just a part of it as it'd be more "correct"
`Unsupported file type conversion. A dictionary of supported input and output file types can be found at API endpoint \'/fileTypes\'`. Even matching on the first sentence would be better as we want to ensure the right message is being returned in the array - there could be cases in the future where there's multiple unsupported messages, and we wouldn't know which type of unsupported it was.